RecoPixelVertexing/PixelVertexFinding/src/PVClusterComparer.cc

```diff
@@ -78,7 +78,7 @@ double PVClusterComparer::pTSquaredSum(const reco::Vertex &v) {
 void PVClusterComparer::setChisquareQuantile() {
   std::vector<double> maxChi2(20, 0.);
   if (track_prob_min_ >= 0. && track_prob_min_ <= 1.)
-    for (size_t ndof = 0; ndof < maxChi2_.size(); ++ndof)
+    for (size_t ndof = 0; ndof < maxChi2.size(); ++ndof)
       // http://root.cern.ch/root/html/TMath.html#TMath:ChisquareQuantile
       maxChi2[ndof] = TMath::ChisquareQuantile(1 - track_prob_min_, ndof);
```

Comment:
why not filling maxChi2_ directly?

Reply:
yes .. it could have been done . I thought i will just make the minor  code correction .. but now that its mentioned , I shall update the PR with the code that fills maxChi2_ directly